src/main/java/pl/sda/eventmanager/services/AdminPanelService.java

```diff
@@ -0,0 +1,24 @@
+package pl.sda.eventmanager.services;
+
+import org.springframework.stereotype.Service;
+import pl.sda.eventmanager.model.User;
+import pl.sda.eventmanager.repositories.UserRepository;
+
+import java.util.List;
+
+@Service
+public class AdminPanelService {
+
+    private final UserRepository userRepository;
+
+    public AdminPanelService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public List<User> findAll(){
```

Comment:
move that method to `UserService` (or remove if there is already method `findAll()`), as pointed below you don't need to exclude admin